tests/api/test_nanopubs.py

```diff
@@ -1,27 +1,22 @@
-from rdflib import *
 
 import json
-
+from rdflib import ConjunctiveGraph, Graph, URIRef
+from rdflib.namespace import RDF
 from whyis.test.api_test_case import ApiTestCase
 
 
 class NanopubTest(ApiTestCase):
 
-    turtle  = '''
+    turtle = '''
 <http://example.com/janedoe> <http://schema.org/jobTitle> "Professor";
     <http://schema.org/name> "Jane Doe" ;
     <http://schema.org/telephone> "(425) 123-4567" ;
     <http://schema.org/url> <http://www.janedoe.com> ;
     <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://schema.org/Person> .
 '''
-    
-    def test_create(self):
-        self.login(*self.create_user("user@example.com","password"))
-
-        response = self.client.post("/pub", data=self.turtle, content_type="text/turtle",follow_redirects=True)
-        
-        self.assertEquals(response.status,'201 CREATED')
 
+    def test_create(self):
+        response = self.publish_nanopub(self.turtle, "text/turtle")
```

Comment:
Use a keyword only declaration

def publish_nanopub(self, *, content_type: str, data: str):
...

Then you have to do data=self.turtle, content_type="text/turtle".

Positional arguments should only be used for single-argument callables (functions, methods, etc.) or very well-known ones in the standard library.